src/pseudopeople/interface.py

```diff
@@ -298,7 +298,9 @@ def generate_decennial_census(
         user_filters.append(
             (DATASETS.census.state_column_name, "==", get_state_abbreviation(state))
         )
-    return _generate_dataset(DATASETS.census, source, seed, config, user_filters, verbose)
+    return _generate_dataset(
```

Comment:
so you're only sorting for census? The others don't need to be sorted?

Reply:
The way we should be implemented this we currently can't - so we are doing it this way for now and will fix with a larger refactor later.